src/Microsoft.AspNet.Mvc.TagHelpers/Internal/GlobbingUrlBuilder.cs

```diff
@@ -137,6 +140,67 @@ private string ResolveMatchedPath(string matchedPath)
             return RequestPathBase.Add(relativePath).ToString();
         }
 
+        private class PathComparer : IComparer<string>
+        {
+            public int Compare(string x, string y)
+            {
+                // < 0 = x < y
```

Comment:
I'd remove comment, you're assuming the reader doesn't already understand the implications of an `IComparer<string>`